torch/_inductor/utils.py

```diff
@@ -1181,12 +1181,31 @@ class Placeholder(enum.Enum):
     #include <c10/cuda/CUDAStream.h>
     #include <torch/csrc/inductor/aot_runtime/interface.h>
 
-    void run(
-            std::vector<at::Tensor>& input_tensors,
-            std::vector<at::Tensor>& output_tensors) {
+    class AOTInductorModelContainer {
```

Comment:
Please use a different name as this can cause confusion with https://github.com/pytorch/pytorch/blob/1427b8149c34b429bb19783045bbc4766bdb55f3/torch/csrc/inductor/aot_runtime/interface.h#L39